docs/02-advanced-configuration.md

```diff
@@ -42,7 +42,45 @@ specified [declarations](http://flowtype.org/docs/declarations.html) when type c
 
 ### `[options]`
 
-The `[options]` heading in a `.flowconfig` file tells `flow` lets you specify the module system via `module.system=haste` or `module.system=node`.
+The `[options]` heading in a `.flowconfig` file can contain several key-value pairs:
+
+- `log.file` (string): the path to the log file (defaults to `/tmp/flow/<escaped root path>.log`)
+- `module.name_mapper` (regex -> string): specify a regular expression to match against module names, and a replacement value, separated by a `->`.
+
+  For example:
+  
+  ```
+  module.name_mapper: '^image![a-zA-Z0-9$_]+$' -> 'ImageStub'
+  ```
+  
+  makes Flow treat `require('image!foo.jpg')` as if it was `require('ImageStub')`.
+  
+  (**note:** you can specify `module.name_mapper` multiple times)
+
+- `module.system` (`node` | `haste`): the module system to use to resolve `import` and `require`
+
+- `munge_underscores` (boolean): set this to `true` to have Flow treat underscore-prefixed class properties and methods as private. This should be used in conjunction with [`jstransform`'s ES6 class transform](https://github.com/facebook/jstransform/blob/master/visitors/es6-class-visitors.js), which enforces the same privacy at runtime.
+
+- `traces` (integer): enables traces on all error output (showing additional details about the flow of types through the system), to the depth specified. This can be very expensive, so is disabled by default.
+
+- `strip_root` (boolean): set this to `true` to always strip the root directory from file paths in error messages
+
+- `suppress_comment` (regex): defines a magical comment that suppresses any Flow errors on the following line. For example:
+
+  ```
+  suppress_comment: \\(.\\|\n\\)*\\$FlowFixMe
+  ```
+  
+  will match a comment like this:
+  
+  ```
+  // $FlowFixMe: suppressing this error until we can refactor
+  var x : string = 123;
+  ```
+  
+  and suppress the error.
```

Comment:
It's worth mentioning in here somewhere that it will also cause an error if there is not actually a Flow error on the following line